playwright/_impl/_page.py

```diff
@@ -605,9 +605,13 @@ async def unroute(
     async def route_from_har(
         self,
         har: Union[Path, str],
-        url: URLMatch = None,
+        url: Union[Pattern, str] = None,
         not_found: RouteFromHarNotFoundPolicy = None,
+        update: bool = None,
     ) -> None:
+        if update:
```

Comment:
Now I wonder whether we should use `recordHar` parameters if they were passed to the context constructor :smile: